src/content/ui/adaptive-responsive/large-screens.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Large screen devices
+description: >
+  Things to keep in mind when adapting apps to large screens.
+short-title: Large screens
+---
+
+<?code-excerpt path-base="ui/adaptive_app_demos"?>
+
+This page provides guidance on optimizing your
+app to improve its behavior on large screens.
+
+Flutter, like Android, defines [large screens][] as tablets,
+foldables, and ChromeOS devices running Android. Flutter
+_also_ defines large screen devices as web, desktop,
+and iPads.
+
+<b>PENDING: Mariam, do you have any specific iPad numbers?</b>
+
+:::secondary Why do large screens matter, in particular?
+Demand for large screens continues to increase.
+As of January 2024,
+more than [270 million active large screen][large screens]
+and foldable devices run on Android.
+When your app supports large screens,
+it also receives other benefits.
+Optimizing your app to fill the screen.
+For example, it:
+
+* Improves your app's user engagement metrics.
+* Increases your app's visibility in the Play Store.
+  Recent [Play Store updates][] show ratings by
+  device type and indicates when an app lacks
+  large screen support. 
+* Ensures that your app meets iPadOS submission
+  guidelines and is [accepted in the App Store][].
+:::
+
+[accepted in the App Store]: https://developer.apple.com/ipados/submit/
+[large screens]: {{site.android-dev}}/guide/topics/large-screens/get-started-with-large-screens
+[Play Store updates]: {{site.android-dev}}/2022/03/helping-users-discover-quality-apps-on.html
+
+## Layout with GridView
+
+Consider the following screenshots of an app.
+The app displays its UI in a `ListView`.
+The image on the left shows the app running
+on a mobile device. The image on the right shows the
+app running on a large screen device
+_before the advice on this page was applied_.
+
+![Sample of large screen](/assets/images/docs/ui/adaptive-responsive/large-screen.png){:width="90%"}
+
+This is not optimal.
+
+The [Android Large Screen App Quality Guidelines][guidelines]
+and the "iOS equivalent"
+say that neither text nor boxes should take up the
+full screen width. How to solve this in an adaptive way?
+
+[guidelines]: https://developer.android.com/docs/quality-guidelines/large-screen-app-quality
+
+The answer is to either use `GridView` or the
+`maxWidth` property of `BoxConstraints`.
```

Comment:
this makes it sound like there would be two sections below this explaining how to do each option. 

instead there is one section on gridview and a couple sentences at the bottom within the Gridview heading about max width.

either we make a separate section for max width with Box constraints or just leave GridView listed here and have the couple sentences we have currently explaining other ways to do it

Reply:
Very good catch. Fixed (the easy way, for now)